.gitignore

```diff
@@ -1,3 +1,5 @@
+requirements*
```

Comment:
this is something the Makefile generates in order to replicate the previous make command behavior for installing specific dependency groups to the active environment instead of a uv managed env.